docs/src/main/asciidoc/security.adoc

```diff
@@ -3,161 +3,185 @@ This guide is maintained in the main Quarkus repository
+For example, the credentials can come from the `Authorization` header, client HTTPS certificates, or cookies.
 
-== Architecture
-
-`HttpAuthenticationMechanism` is the main entry into Quarkus HTTP Security.
-
-Quarkus Security Manager uses `HttpAuthenticationMechanism` to extract the authentication credentials from the HTTP request and delegates to `IdentityProvider` to
-complete the conversion of these credentials to `SecurityIdentity`.
-
-For example, the credentials may be coming with the HTTP `Authorization` header, client HTTPS certificates or cookies.
-
-`IdentityProvider` verifies the authentication credentials and maps them to `SecurityIdentity` which contains the username, roles, the original authentication credentials, and other attributes.
+`IdentityProvider` verifies the authentication credentials and maps them to `SecurityIdentity`, which has the username, roles, original authentication credentials, and other attributes.
 
 For every authenticated resource, you can inject a `SecurityIdentity` instance to get the authenticated identity information.
 
-In some other contexts you may have other parallel representations of the same information (or parts of it) such as `SecurityContext`
-for JAX-RS or `JsonWebToken` for JWT.
+In other contexts, it is possible to have other parallel representations of the same information or parts of it, for example, `SecurityContext`
+for JAX-RS or `JsonWebToken` for JSON Web Tokens (JWT).
 
-== Authentication mechanisms
+== Getting started with Quarkus Security
 
-Quarkus supports several sources to load authentication information from.
+Before you start building security into your Quarkus applications, review the overview information to learn about the different authentication and authorization mechanisms that Quarkus supports.
 
-=== Basic and Form Authentication Mechanisms
+To get started with security in Quarkus, we recommend that you first combine the Quarkus built-in xref:security-basic-auth-concept.adoc[Basic HTTP authentication] with the JPA identity provider to enable role-based access control (RBAC). 
+Complete the steps in the ref:security-getting-started.adoc[Secure a Quarkus application with Basic authentication] tutorial.
+After you have successfully secured your Quarkus application with basic HTTP authentication, you can increase the security further by adding more advanced authentication mechanisms, for example, OpenID Connect (OIDC) authentication.
 
-Basic and Form HTTP-based authentication mechanisms are the core authentication mechanisms supported in Quarkus.
-Please see xref:security-basic-auth-concept.adoc[Basic HTTP Authentication] and xref:security-built-in-authentication.adoc#form-auth[Form HTTP Authentication] for more information.
+== Supported authentication mechanisms in Quarkus Security
```

Comment:
IMHO `==AuthenticationMechanisms` is simpler

Reply:
Gotcha :+1: